Update GetZoneFromNameId and GetNameOfZone

LokiLeiche opened the pull request:
These changes are based on Issue #1246. 

**GetZoneFromNameId**
In this native the docs page included a list of the zone abbreviations and their full names which is completely irrelevant for this native. It takes a zone string ID as listed in the firts column of `update\update.rpf\common\data\levels\gta5\popzone.ipl` as argument and returns the zone ID as integer, which is just the index of that zone in the popzone.ipl starting at 1.
The list being there is misleading as it causes people to think, the native converts the abreviations to the full are name which seemingly is also what the issue author thought but just isn't true.

**GetNameOfZone**
As the issue author stated, 3 zones that are listed in the popzone.ipl file are missing here, so I added them. Credits to @MichaelCoding25 for listing these. I verified these zones exist and work by making sure they are returned by the native when in the area configured by popzone.ipl. The issue author also said the zone `SANAND` should be removed since it's never used anywhere but I couldn't verify that so I left it for now.

Both of these have basically not been touched since the initial commit 7 years ago so they were probably never properly manually checked.

MichaelCoding25 commented on ZONE/GetNameOfZone.md:
```diff
@@ -64,6 +66,7 @@ MTJOSE = Mount Josiah
 MURRI = Murrieta Heights  
 NCHU = North Chumash  
 NOOSE = N.O.O.S.E  
+NOOSE = Galileo Observatory  
```
Should be OBSERV and not NOOSE

LokiLeiche replied:
Ah yeah, seems like I forgot to replace that after copy pasting... Sorry and thanks for pointing it out

MichaelCoding25 commented:
Looks good and thanks for actually doing this.
One thing, maybe it's better in the ZONE/GetZoneFromNameId.md to have a list of these? because I feel most people won't wanna have to get software like openiv to get the actual file

LokiLeiche commented:
> Looks good and thanks for actually doing this. One thing, maybe it's better in the ZONE/GetZoneFromNameId.md to have a list of these? because I feel most people won't wanna have to get software like openiv to get the actual file

Thanks to you for doing the research for it. not sure about the list in GetZoneFromNameId. If you mean the old list that I removed, I feel like it's unnecessary since that is not the zone name used in the native so it's kinda misleading. If you mean a new zone list like the gist you had in your issue, that's way too long for adding it to the native documentation directly, but maybe link the gist or a pastebin with that list? idk

MichaelCoding25 commented:
> > Looks good and thanks for actually doing this. One thing, maybe it's better in the ZONE/GetZoneFromNameId.md to have a list of these? because I feel most people won't wanna have to get software like openiv to get the actual file
> 
> Thanks to you for doing the research for it. not sure about the list in GetZoneFromNameId. If you mean the old list that I removed, I feel like it's unnecessary since that is not the zone name used in the native so it's kinda misleading. If you mean a new zone list like the gist you had in your issue, that's way too long for adding it to the native documentation directly, but maybe link the gist or a pastebin with that list? idk

Yeah Maybe using the gist or making a pastebin would be good. I just updated the gist to include the ids and descriptions of the zones

LokiLeiche commented:
Thanks @technetium-cfx for the review, I opened a PR in fivem-docs where I added the list as game reference and updated the link to it here, therefore only merge this with my new PR. https://github.com/citizenfx/fivem-docs/pull/554#issue-3335366242

If there's anything else, let me know.
